config.md

```diff
@@ -340,6 +340,52 @@ For Linux-based systems, the `process` object supports the following process-spe
 
     * **`class`** (string, REQUIRED) specifies the I/O scheduling class. Possible values are `IOPRIO_CLASS_RT`, `IOPRIO_CLASS_BE`, and `IOPRIO_CLASS_IDLE`.
     * **`priority`** (int, REQUIRED) specifies the priority level within the class. The value should be an integer ranging from 0 (highest) to 7 (lowest).
+* **`landlock`** (object, OPTIONAL) specifies the Landlock unprivileged access control settings for the container process.
+    Note that `noNewPrivileges` must be set to true to use this feature.
+    For more information about Landlock, see [Landlock documentation][landlock].
+    `landlock` contains the following properties:
+
+    * **`ruleset`** (object, OPTIONAL) the `ruleset` field identifies a set of rules (i.e., actions on objects) that need to be handled (i.e., restricted).
+    The `ruleset` currently contains the following types:
+        * **`handledAccessFS`** (array of strings, OPTIONAL) is an array of FS typed actions that are handled by a ruleset.
+        If no rule explicitly allow them, they should then be forbidden.
+        * **`handledAssessNetwork`** (array of strings, OPTIONAL) is an array of NETWORK typed actions that are handled by a ruleset. (The NETWORK typed actions are avaliable when the ABI version >= 4. the behavior of the NETWORK typed actions is not used when the ABI version is less than 4 will depend on the **`disableBestEffort`**)
+    * **`rules`** (object, OPTIONAL) the `rules` field specifies the security policies (i.e., actions allowed on objects) to be added to an existing ruleset.
+    The `rules` currently contains the following types:
+        * **`pathBeneath`** (array of objects, OPTIONAL) is an array of the file-hierarchy typed rules.
+        Entries in the array contain the following properties:
+            * **`allowedAccess`** (array of strings, OPTIONAL) is an array of FS typed actions that are allowed by a rule. The actions are grouped by the ABI version in the following description:
+                1. ABI version >= 1:
+                    1. exectute
+                    2. write_file
+                    3. read_file
+                    4. read_dir
+                    5. remove_dir
+                    6. remove_file
+                    7. make_char
+                    8. make_dir
+                    9. make_reg
+                    10. make_sock
+                    11. make_fifo
+                    12. make_block
+                    13. make_sym
+                2. ABI version >= 2:
+                    1. refer
+                3. ABI version >= 3:
+                    1. truncate
+            * **`paths`** (array of strings, OPTIONAL) is an array of files or parent directories of the file hierarchies to restrict.
+        * **`portBeneath`** (array of objects, OPTIONAL) is an array of the network-hierarchy typed rules.
```

Comment:
portBeneath? Is this for all ports below the number? Or is this just a c&p mistake?

Reply:
Yes, it's for all ports below the number. Actually I follow style with `pathBeneath`

Reply:
Hmm, okay, thanks. I'm in an off-site this week, so I'll check this up next week.

Reply:
I can't make sense out of that.

If I want to have a rule for port 21, 22, 53 and 80, it seems super complicated to have a portBeneath rule. For filesystems that makes sense, because is all beneath this (i.e. subdirs and everything below that), this is applied.

But what does it mean for ports? It seems like a very complicated way to express it.

Oh, it seems @gnoack already commented on this. I fully agree: https://github.com/opencontainers/runtime-spec/pull/1241/files#r1508607132

Reply:
Thanks, exactly -- this is still confusing and would better be named "net port" [in line with the `landlock.h` header file](https://github.com/torvalds/linux/blob/master/include/uapi/linux/landlock.h#L98).

It does *not* permit access for all ports below a given number, but only for that specific port number itself, and the word "hierarchy" is also misleading, as there is nothing hierarchical about that (probably copied over from the file system access rights documentation).